xml/obs/systemsmanagement:Agama:Staging.xml

```diff
@@ -0,0 +1,15 @@
+<openQA
+    project_pattern="systemsmanagement:Agama:Staging"
+    dist_path="obspublish-other::openqa/systemsmanagement:Agama:Staging/"
+    version="agama-0.9"
```

Comment:
I don't think this could have any meaning for a staging project, but I'm not sure what to use ... I guess this is necessary for openQA to configure the mediums, perhaps, `agama-staging` would be more correct.